.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: kind/bug
+assignees: ''
+
+---
+
+**Bug Description**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**System Info**
+Add information about your system. If Youki is compiling and running on your system, please add o/p of `youki info` .
+
+**Setup Info**
+If you are experiencing issues in compiling Youki, or related deps, please add details regarding your setup, dependencies etc. Also if you are running Youki via something else (Docker, podman, k8s) add its info here.
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
Is it really necessary to have all this information to create an issue? I feel that it'd be a waste to miss the opportunity to have an issue created at this stage of the phase. But, it might be useful to have labels and so on.

Reply:
> I feel that it'd be a waste to miss the opportunity to have an issue created at this stage of the phase

I'm not sure what you mean here :sweat_smile: 

As for the details, I think we can also add an empty issue template, which will not have any fields, and behave similar to how current 'new issue' works. I agree that having to add this much info can be troublesome, but I feel having these details would be good for understanding and debugging the issues.

Reply:
Sorry, let me tell you more details. 

> but I feel having these details would be good for understanding and debugging the issues.

Yes, this helps **us** understand the issue. However, how about an issue's author?  Perhaps filling in some fields is tedious, and they may stop making issues. I'm worried about it.
But, if we use the issue form, we can mark require or not require. We use it in order to solve this problem.

Reply:
Hey, yep I get it now. That is a valid concern. I'll update the  PR to use forms instead.